docs/FAQs.md

```diff
@@ -21,11 +21,16 @@ Provisioners are heterogeneous, which means that the nodes they manage are sprea
 ### What if my pod is schedulable for multiple Provisioners?
 It's possible that an unconstrained pods could flexibly schedule in multiple groups. In this case, Provisioners will race to create a scheduling lease for the pod before launching new nodes, which avoids unnecessary scale out.
 ## Reallocation
-TODO
+### How does Karpenter terminate nodes?
+Karpenter annotates nodes that are underutilized with a TTL. If the node remains underutilized after the TTL expires, Karpenter then cordons the node and uses the Kubernetes Eviction API to evict all non-daemonset pods. Once the node is empty, the node is terminated.
+### How does Karpenter decide which nodes it can terminate? 
+A provisioner will only take action on nodes that it manages. This means that a node will only be considered for termination if it labeled underutilized by the provisioner that manages it.
+### How do I know if a node is underutilized?
+Nodes are considered underutilized if they have 0 non-daemon set pods scheduled. We plan to include more use cases in the future.
```

Comment:
It might make sense to talk about time here. I assume we're not suggesting some kind of instantaneous termination the moment it hits 0? Is there a minimum time a node has to be at 0 before being considered underutilized?

Reply:
I think a better organization of these questions would help answer this. I just wanted to say this is when it's underutilized, where it would be considered for termination after a period of time. I can understand the confusion though.